packages/admin-portal/src/core/connect-session.ts

```diff
@@ -0,0 +1,12 @@
+import { ReapitConnectBrowserSession } from '@reapit/connect-session'
+// Referencing config direct rather than via window because window.reapit.config not available at compile time
+// and throws a runtime error
+import config from '../../config.json'
```

Comment:
We can use window.reapit.config now - I worked this out. You just need to refactor the run function in the index.tsx to lazy load the App when the config has been set - see https://github.com/reapit/foundations/blob/master/packages/aml-checklist/src/core/index.tsx#L61